### PR TITLE
CRAYSAT-1480: Fix ordering of NCN power operations

### PR DESCRIPTION
**Pull request by jack-stanek-hpe**
## Summary and Scope

The ordering of NCN power operations in the boot phase of `sat bootsys` was fixed. See Ryan's comment [here](https://github.com/Cray-HPE/sat/pull/36#discussion_r949647100).

## Issues and Related PRs

* Resolves [CRAYSAT-1480](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1480)

### Tested on:

  * Local development environment

### Test description:

Build man pages and inspect.

## Risks and Mitigations

N/A.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by ekamin-hpe — APPROVED** [review]
My only question would be if this needs a change log entry. It is nearly trivial and we're really only supposed to document "notable" changes in that file, so I'm leaning no.

The only issue would be around the next stable build having an incremented version number, and I think it will regardless, so I don't see that as being an issue.

**Comment by haasken-hpe** [comment]
> My only question would be if this needs a change log entry. It is nearly trivial and we're really only supposed to document "notable" changes in that file, so I'm leaning no.
> 
> The only issue would be around the next stable build having an incremented version number, and I think it will regardless, so I don't see that as being an issue.

It's probably slightly easier and safer to just have a changelog entry here. Not going to hold up my approval though. I'm assuming something else will go in (probably that [CRAYSAT-1536](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1536) bugfix), and that will cause us to create an "Unreleased" changelog section.

**Review by haasken-hpe — APPROVED** [review]

**Comment by jack-stanek-hpe** [comment]
> > My only question would be if this needs a change log entry. It is nearly trivial and we're really only supposed to document "notable" changes in that file, so I'm leaning no.
> > The only issue would be around the next stable build having an incremented version number, and I think it will regardless, so I don't see that as being an issue.
> 
> It's probably slightly easier and safer to just have a changelog entry here. Not going to hold up my approval though. I'm assuming something else will go in (probably that [CRAYSAT-1536](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1536) bugfix), and that will cause us to create an "Unreleased" changelog section.

I went ahead and created a changelog entry and `[Unreleased]` section. I'll merge this once CI completes.

**Pull request merged**
